src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -87,6 +87,12 @@ module sonic-vlan {
 						enum disable;
 					}
 				}
+
+				leaf static-anycast-gateway {
+					description "Enable/disable static anycast gateway for the vlan interface";
```

Comment:
Where is the SAG GW IP field?

Reply:
The IP address is configured on the VLAN interface, no additional field needs to be added.

Reply:
Ok